src/common/components/profile-menu/index.tsx

```diff
@@ -46,15 +47,32 @@ export class ProfileMenu extends Component<Props> {
             label: ProfileFilter[section] ? _t(`profile.section-${section}`) : '',
             items: [
                 ...[ProfileFilter.blog, ProfileFilter.posts, ProfileFilter.comments, ProfileFilter.replies].map((x) => {
+                    let isActive = false;
+                        if(section===x && section ==='blog' && data.length!==0){
```

Comment:
please keep consistent spacing, `section===x` and `section ==='blog'` and `data.length!==0` all using different styling.